config/main.py

```diff
@@ -295,6 +295,10 @@ def _stop_services():
         'hostcfgd',
     ]
 
+    # on Mellanox platform pmon is stopped by syncd
```

Comment:
shall we assign different elements set to "service_to_stop" based on the "asic_type" instead of removing some element here? it seems like a hack.